app/assess/models/ui/applicants_response.py

```diff
@@ -253,7 +253,7 @@ def _ui_component_from_factory(item: dict, application_id: str):
 
         return NewAddAnotherTable.from_dict(item)
 
-    elif presentation_type in ("text", "list"):
+    elif presentation_type in ("text", "list", "free_text"):
```

Comment:
@ramsharma-prog You may need to revert this if the work you're doing to render the html is different FYI

Reply:
thanks Ram, i guess we merge your change to render free text and i can rebase onto it\?

Reply:
Sounds good @Adam-W1

Reply:
PR: https://github.com/communitiesuk/funding-service-design-assessment/pull/266/files